cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -2,7 +2,10 @@
 Distributed training using Horovod on Spark
 -------------------------------------------
 """
-
+# %%
+# Importing the Libraries
+# ========================
+# The required packages are imported into the environment.
```

Comment:
Can we write the content in the present tense? Like, say, "Let's import all the required packages into the environment". What do you say?

Reply:
Sure, I will do that